tests/e2e/helpers/api/api-database.ts

```diff
@@ -0,0 +1,107 @@
+
+import { t } from 'testcafe';
+import * as request from 'supertest';
+import { asyncFilter, doAsyncStuff } from '../async-helper'
+import { AddNewDatabaseParameters, OSSClusterParameters } from '../../pageObjects/add-redis-database-page';
+import { apiUrl } from '../../helpers/conf';
+
+const endpoint = apiUrl;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'; // lgtm[js/disabling-certificate-validation]
+
+/**
+ * Add a new Standalone database through api using host and port
+ * @param databaseParameters The database parameters
+ */
+export async function addNewStandaloneDatabase(databaseParameters: AddNewDatabaseParameters): Promise<void> {
+    const response = await request(endpoint).post(`/instance`)
+        .send({ "name": databaseParameters.databaseName, "host": databaseParameters.host, "port": databaseParameters.port })
+        .set('Accept', 'application/json');
+
+    await t.expect(await response.status).eql(201);
+    await t.expect(await response.body.name).eql(databaseParameters.databaseName);
+}
+
+/**
+ * Add a new Standalone databases through api using host and port
+ * @param databasesParameters The databases parameters array
+ */
+export async function addNewStandaloneDatabases(databasesParameters: any): Promise<void> {
```

Comment:
Why type `any`? You use with typescript, you can create your own type

Reply:
fixed, thanks